packages/rspack/src/config/zod/output.ts

```diff
@@ -0,0 +1,93 @@
+import { z } from "zod";
+
+function chunkLoadingType() {
+	return z
+		.enum(["jsonp", "import-scripts", "require", "async-node", "import"])
+		.or(z.string());
+}
+
+function wasmLoadingType() {
+	return z.enum(["...", "fetch-streaming", "fetch", "async-node"]);
+}
+
+function libraryType() {
+	return z.enum([
+		"...",
```

Comment:
What's this?

Reply:
https://github.com/web-infra-dev/rspack/blob/f20a34d3c9892ef1e2070f208c932dfd7c3bdfe2/packages/rspack/src/config/normalization.ts#L102-L103